src/ui/src/components/chart/AreaChart.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {
+  JinjatECharts,
+  JinjatEChartsProps,
+} from "@components/chart/JinjatECharts";
+
+interface JinjatAreaChartsProps extends JinjatEChartsProps {
+  data: undefined;
+  x: undefined;
```

Comment:
let's add types for these arguments

Reply:
In Evidence's code, they all are given as undefined. I also had doubts about it. Is there any other source where I could find the actually expected types of these arguments?